tests/covdev_test.cpp

```diff
@@ -60,7 +64,7 @@ void init(int argc, char **argv)
 
   ColorSpinorParam csParam;
   csParam.nColor = nColor;
-  csParam.nSpin = 4;
+  csParam.nSpin = test_type == 0 ? 4 : 1; // use --test 1 for staggered case
```

Comment:
Not a hill I'll die on, but we should put in an `errorQuda` if the test isn't 1 or 4, just because it's quick.

Reply:
addressed in 6323dc231273fc398ec7c6115fb551d519656705